### PR TITLE
New requirements for passwordless auth

### PR DESCRIPTION
**Pull request by Lars1963**
closes https://github.com/nextcloud/server/issues/20600

Signed-off-by: Lars van Ravenzwaaij ict@ravenzwaaij.eu

### CODE REVIEW

**Comment by Lars1963** [comment]
@rullzer 
Are there other places to mention this?
would you like me to update the examples also?

**Comment by kesselb** [comment]
I think the modules should be added to `Required for specific apps`. They are only required for passwordless login: https://github.com/nextcloud/server/pull/19858/files#diff-d9cc35548ebcf7fc4d2eb9681d2f2de9R254

**Comment by Lars1963** [comment]
@kesselb 
So better?

File: admin_manual/installation/source_installation.rst [hunk 1]
```diff
@@ -73,11 +73,12 @@ Required for specific apps:
   :doc:`../configuration_files/external_storage/smb`)
 * PHP module ftp (for FTP storage / external user authentication)
 * PHP module imap (for external user authentication)
+* PHP module bcmath (for  passwordless login)
```
Comment (MorrisJobke) [inline admin_manual/installation/source_installation.rst:76]:
```suggestion
* PHP module bcmath (for passwordless login)
```

**Pull request merged**